package.json

```diff
@@ -13,22 +13,22 @@
     "./use-at-your-own-risk": "./lib/unsupported-api.js"
   },
   "scripts": {
+    "build:docs:update-links": "node tools/fetch-docs-links.js",
+    "build:release": "node Makefile.js generateRelease",
+    "build:release:alpha": "node Makefile.js generatePrerelease -- alpha",
+    "build:release:beta": "node Makefile.js generatePrerelease -- beta",
+    "build:release:publish": "node Makefile.js publishRelease",
+    "build:release:rc": "node Makefile.js generatePrerelease -- rc",
+    "build:site": "node Makefile.js gensite",
+    "build:webpack": "node Makefile.js webpack",
+    "lint": "node Makefile.js lint",
+    "lint:fix": "node Makefile.js lint -- fix",
+    "lint:docs:js": "node Makefile.js lintDocsJS",
+    "lint:docs:js:fix": "node Makefile.js lintDocsJS -- fix",
```

Comment:
This is not alphabetical because "lint:f" should come after "lint:d". If we write a validator at some point we'll need to fuss over these details.